app/services/decisions/client_decision_tree.rb

```diff
@@ -133,37 +129,22 @@ def start_address_journey(address_class)
     def after_has_nino
       if current_crime_application.not_means_tested?
         edit('/steps/case/urn')
-      elsif FeatureFlags.partner_journey.enabled?
-        edit(:has_partner)
       else
-        edit('/steps/dwp/benefit_type')
+        edit(:has_partner)
       end
     end
 
     def after_has_partner
-      if start_partner_journey? && FeatureFlags.partner_journey.enabled?
+      if form_object.client_has_partner.yes?
         edit('/steps/partner/relationship')
-      elsif form_object.client_has_partner.no? && FeatureFlags.partner_journey.enabled?
-        edit(:relationship_status)
-      elsif form_object.client_has_partner.yes?
-        show(:partner_exit)
       else
-        edit(:details)
+        edit(:relationship_status)
       end
     end
 
     def applicant
       @applicant ||= current_crime_application.applicant
     end
-
-    # TODO: Consider whether !crime_application.age_passported? is more appropriate?
-    def start_partner_journey?
-      return false unless FeatureFlags.partner_journey.enabled?
-
-      form_object.client_has_partner.yes? &&
-        !current_crime_application.not_means_tested? &&
-        !applicant.under18?
-    end
```

Comment:
What's the reason we no long care about u18 etc? Is it because we would not get to this point now the position of the has_partner question has moved?

Reply:
That's right, has partner question won't be shown to u18 or non-means applicants